hub/examples/image_retraining/label_image.py

```diff
@@ -41,67 +42,26 @@
 
 def wiki(celestial_object):
     ans = celestial_object
+    with open(os.path.join(r'examples/image_retraining/display_info.yaml'), 'r') as stream:
```

Comment:
path needs to be corrected as it will be DIRECTORY_PATH/Celestial-bodies-detection/hub/examples/image_retraining/

Reply:
Got cwd and added hub/examples/image_retraining to it.

Reply:
That's wrong as cwd will be `/home/uesrname/git/Celestial-bodies-detection/hub/examples/image_retraining/` for other files to work